workflow-templates/release-go-crosscompile-task.md

```diff
@@ -0,0 +1,110 @@
+# "Release" workflow (Go, Task)
```

Comment:
I suggest differentiating this title in some way from the one in the `workflow-templates/release-go-task.md` file. That could include changing the title of  `workflow-templates/release-go-task.md`.